documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -1589,6 +1589,12 @@ ifdef::product[]
 The {prodname} Oracle connector is compatible with Oracle installed as a standalone instance.
 endif::product[]
 
+=== Built-in excluded schemas
+
+The {prodname} Oracle connector automatically excludes capturing changes from the following schemas:
+`appqossys`, `audsys`, `ctxsys`, `dvsys`, `dbsfwuser`, `dbsnmp`, `qsmadmin_internal`, `lbacsys`, `mdsys`, `ojvmsys`, `olapsys`, `orddata`, `ordsys`, `outln`, `sys`, `system`, `wmsys`, and `xdb`.
+In order to guarantee that a table is captured by the connector, please be sure to any other schema.
```

Comment:
To enable the connector to capture changes from a table, the table must use a schema that is not named in the preceding list.